### DISCUSSION TITLE
custom keyboard shortcuts

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [searxng/searxng#3465] [Q&A] [answered]
I use an alternative keyboard layout and would like to change hjkl to something like haei instead.

**Comment by azizsayfiddin** [comment]
is it possible?

**Comment by return42** [comment, marked as answer]
> custom keyboard shortcuts ... is it possible?

no

**Reply by azizsayfiddin** [reply]
Can I change them if I host my own instance of searxng?
How easy is it to implement? Can it be done via some config file or such, or is it hard-coded? I want to use the same keys everywhere.

**Reply by return42** [reply]
https://github.com/searxng/searxng/blob/master/client/simple/src/js/main/keyboard.ts

**Reply by azizsayfiddin** [reply]
thank you
